twisted/web/test/test_http2.py

```diff
@@ -1403,6 +1403,32 @@ def validate(streamID):
         return a._streamCleanupCallbacks[1].addCallback(validate)
 
 
+    def test_resetAfterBody(self):
+        """
+        A client that immediately resets after sending the body causes Twisted
+        to send no response.
+        """
+        f = FrameFactory()
```

Comment:
Understood.

minor comment: f is a very generic name ... it can be f for frame, frameFactory httpFactory :)